clients/src/main/java/org/apache/kafka/clients/consumer/Consumer.java

```diff
@@ -95,8 +95,14 @@
     /**
      * @see KafkaConsumer#commitSync(Map)
      */
+    @Deprecated
```

Comment:
According to the KIP, the only methods to be deprecated are `close(TimeUnit, long)` and `poll(long)`.